integration-tests/kafka/src/main/resources/application.properties

```diff
@@ -5,6 +5,7 @@ quarkus.log.category.\"org.apache.zookeeper\".level=WARN
 quarkus.kafka-streams.bootstrap-servers=localhost:19092
 quarkus.kafka-streams.application-id=streams-test-pipeline
 quarkus.kafka-streams.topics=streams-test-categories,streams-test-customers
+quarkus.kafka-streams.health.enabled=true
```

Comment:
This is the default so it should not be specified

Reply:
You're right